sw/host/tests/chip/i2c_target/src/main.rs

```diff
@@ -114,7 +115,7 @@ fn test_wakeup_normal_sleep(opts: &Opts, transport: &TransportWrapper, address:
     test_read_transaction(opts, transport, address)
 }
 
-fn test_wakeup_deep_sleep(opts: &Opts, transport: &TransportWrapper, address: u8) -> Result<()> {
+fn test_wakeup_deep_sleep(opts: &Opts, transport: &TransportWrapper, address: u8, instance:u8) -> Result<()> {
```

Comment:
needs to run rustfmt

Reply:
I always run the clang-format but forgot that I touched rust code. :smile: